misc/bazel/platforms/BUILD.bazel

```diff
@@ -0,0 +1,66 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
I don't enjoy that we now have the platform definitions externally, but the toolchain defs internally (and the toolchains themselves belong to and need to stay internally AFAIUI).
Is there any way we can put this file back to the internal repo?

Reply:
I didn't mind too much, as this is defining constraints, not the actual toolchains. But in any case, this will go